backend/backend/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "BackendContext": "Server=.\\SQLEXPRESS;Database=WeDontByte;Trusted_Connection=True;"
+    "BackendContext": "Server=.\\SQLEXPRESS,1433;Database=WeDontByte;User=sa;Password=Password@123;"
```

Comment:
This throws exceptions for me. I created a separate user and it works if you do not provide the port. For SQLEXPRESS 1433 is the default port. Is is really necessary to give it as a param in connection string?

Reply:
https://social.msdn.microsoft.com/Forums/sqlserver/en-US/e1082cad-9248-44ed-bfcc-c08d7cc08831/login-failed-for-user-sa-microsoft-sql-server-error-18456-severity-14-state-1-when-i-use


Check this link. The port is mandatory for Docker, it seems

Reply:
Reverted the change completely, should work without hiccups in Windows